in_toto/in_toto_record.py

```diff
@@ -145,8 +147,16 @@ def main():
   in_toto_args.add_argument("-k", "--key", type=str, required=True,
       help="Path to private key to sign link metadata (PEM)")
 
-  in_toto_args.add_argument("-v", "--verbose", dest='verbose',
-      help="Verbose execution.", default=False, action='store_true')
+  in_toto_args.add_argument("-v", "--verbose", dest="loglevel",
+      help="Verbose execution.", default=log.logging.WARNING, const=log.logging.INFO,
+      action="store_const")
+
+  in_toto_args.add_argument("-d", "--debug", dest="loglevel",
+      help="Debug statement execution.", default=log.logging.WARNING, const=log.logging.DEBUG,
+      action="store_const")
+
+  in_toto_args.add_argument("-c", "--color", dest="colorize",
+      help="Colorizes output.", default=False, action="store_true")
```

Comment:
I'd rather not have too many new command line options. What do you think of providing multiple `-v` options?
i.e.: `-v` for `INFO` and  `-vv` for `DEBUG`

Reply:
Personally, I think it's more intuitive to leave --debug for debugging, and that's the implementation i've seen with some other software. 

I'm not entirely sure who's right here, but in my personal opinion, I think `-vv` isn't as intuitive.